native/java/org/jpype/proxy/JPypeProxy.java

```diff
@@ -35,6 +36,7 @@ public class JPypeProxy implements InvocationHandler
   public long cleanup;
   Class<?>[] interfaces;
   ClassLoader cl = ClassLoader.getSystemClassLoader();
+  public static Object missing = new Object();
```

Comment:
This static object will be passed back to indicate that Python did not implement the method.   There is nothing special about it other than being an object that the user is unlikely to ever return from a valid proxy.   The object was left public so that I can pass it during testing to trigger the not implemented logic if needed.

Reply:
you say a "valid" proxy. Is a proxy encapsulating a plain Object "invalid"? I'd define a object subclass Undefined, but maybe that is overkill and not even needed.

Reply:
In the rare case that someone implemented a proxy, but somehow managed to not implement a required method for the interface, then the proxy would not be valid.   Thus we need to have the equivalent of "NotImplemented" as a message from the interface.